{"uid":"4b4757e66a1912dae1a509f688f20b0f","name":"categories","children":[{"name":"Product defects","children":[{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 433, in set_control_state\n    self.capture.set_capture_state(request_payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/capture.py\", line 495, in set_capture_state\n    self._start_capture()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/capture.py\", line 524, in _start_capture\n    self._api._ixnetwork.StartCapture()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/ixnetwork.py\", line 2884, in StartCapture\n    return self._execute(\"startCapture\", payload=payload, response_object=None)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 452, in _execute\n    response = self._connection._execute(url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 289, in _execute\n    return self._send_recv(\"POST\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 572, in _send_recv\n    return self._poll()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 640, in _poll\n    return self._process_response_status_code(\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 476, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n  The capture can't be started on a virtual port.","children":[{"name":"test_traffic_custom_header","uid":"769f3a2cd375058c","parentUid":"925eb53b702459181baa2d0323af853d","status":"failed","time":{"start":1741764418062,"stop":1741764431189,"duration":13127},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"test_traffic_transmit_state","uid":"731bde3614294553","parentUid":"925eb53b702459181baa2d0323af853d","status":"failed","time":{"start":1741764418062,"stop":1741764429205,"duration":11143},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"test_udp_header_with_fixed_length_checksum_e2e","uid":"4980e5b38daa29ad","parentUid":"925eb53b702459181baa2d0323af853d","status":"failed","time":{"start":1741764418062,"stop":1741764428769,"duration":10707},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"test_udp_header_with_list_e2e","uid":"b5ce39e0a34e8d6c","parentUid":"925eb53b702459181baa2d0323af853d","status":"failed","time":{"start":1741764418062,"stop":1741764441859,"duration":23797},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"test_udp_header_with_counter_e2e","uid":"be8b9c41b5d47425","parentUid":"925eb53b702459181baa2d0323af853d","status":"failed","time":{"start":1741764418062,"stop":1741764460700,"duration":42638},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"test_traffic_ethernet_pause","uid":"340d6f466d3e0780","parentUid":"925eb53b702459181baa2d0323af853d","status":"failed","time":{"start":1741764418062,"stop":1741764429610,"duration":11548},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"test_traffic_gptv_config","uid":"f0cc891223dcf830","parentUid":"925eb53b702459181baa2d0323af853d","status":"failed","time":{"start":1741764418062,"stop":1741764427781,"duration":9719},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"925eb53b702459181baa2d0323af853d"},{"name":"assert 0 == 1\n +  where 0 = len(<ixnetwork_restpy.testplatform.sessions.ixnetwork.availablehardware.chassis.chassis.Chassis object at 0x7fbe86f371c0>)","children":[{"name":"test_counter_ip_dscp","uid":"f6a059e22d3ca875","parentUid":"a43aacc3f695f034e146de6965af8927","status":"failed","time":{"start":1741764418062,"stop":1741764440962,"duration":22900},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"a43aacc3f695f034e146de6965af8927"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 450, in set_control_state\n    self.traffic_item.transmit(request_payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/trafficitem.py\", line 1307, in transmit\n    self._api._traffic.Apply()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/traffic/traffic.py\", line 1395, in Apply\n    return self._execute(\"apply\", payload=payload, response_object=None)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 452, in _execute\n    response = self._connection._execute(url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 289, in _execute\n    return self._send_recv(\"POST\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 572, in _send_recv\n    return self._poll()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 640, in _poll\n    return self._process_response_status_code(\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 476, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n  Error in L2/L3 Traffic Apply","children":[{"name":"test_update_flows","uid":"a889678df7acbbb2","parentUid":"ef6804d4159127be406a6111d9b6a662","status":"failed","time":{"start":1741764418062,"stop":1741764478437,"duration":60375},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"test_stats_filter","uid":"1415faa690dfa981","parentUid":"ef6804d4159127be406a6111d9b6a662","status":"failed","time":{"start":1741764418062,"stop":1741764487866,"duration":69804},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"ef6804d4159127be406a6111d9b6a662"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 442, in set_control_state\n    self.ngpf.set_protocol_state(request_payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/device/ngpf.py\", line 256, in set_protocol_state\n    self.api._ixnetwork.StartAllProtocols(\"sync\")\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/ixnetwork.py\", line 2862, in StartAllProtocols\n    return self._execute(\"startAllProtocols\", payload=payload, response_object=None)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 452, in _execute\n    response = self._connection._execute(url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 289, in _execute\n    return self._send_recv(\"POST\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 572, in _send_recv\n    return self._poll()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 640, in _poll\n    return self._process_response_status_code(\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 476, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n  Port 'novus1-715849.ccu.is.keysight.com;1;2' in the Port Group 'rx' already used by Port Group 'rxp1'","children":[{"name":"test_vxlan_b2b_v6","uid":"8ac5d62324dc2c83","parentUid":"671d1b8f5c72601d4107f6f26edb5011","status":"failed","time":{"start":1741764418062,"stop":1741764436049,"duration":17987},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"test_vxlan_b2b","uid":"22f14e986a3ad3e8","parentUid":"671d1b8f5c72601d4107f6f26edb5011","status":"failed","time":{"start":1741764418062,"stop":1741764469940,"duration":51878},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"test_bgpv6_routes","uid":"5853acfc93992889","parentUid":"671d1b8f5c72601d4107f6f26edb5011","status":"failed","time":{"start":1741764418062,"stop":1741764471271,"duration":53209},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"671d1b8f5c72601d4107f6f26edb5011"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 276, in set_config\n    self.config_ixnetwork(config)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 361, in config_ixnetwork\n    self.lag.config()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/lag.py\", line 102, in config\n    self._protocol_config()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/lag.py\", line 311, in _protocol_config\n    ixn_static.remove()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/lag/lagportstaticlag_91864be5b58104e6349e12f415e28075.py\", line 259, in remove\n    self._delete()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 441, in _delete\n    raise e\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 438, in _delete\n    self._connection._delete(url)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 286, in _delete\n    return self._send_recv(\"DELETE\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 596, in _send_recv\n    self._process_response_status_code(url, headers, response)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 476, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n Unable to delete /lag:6/protocolStack/ethernet:1/lagportstaticlag:1 Ethernet 1(LAG) 1: Removing a Protocol from a started Protocol stack is not permitted.\n\tCurrent Server Errors/Warnings:\n\t03/12/2025 19:38:00 [WARNING] [Protocols Version Mismatch] Protocols Version Mismatch.\nIxNetwork Protocols Version: 10.80.2412.12\nChassis Protocols Version: 10.50.2410.2\n\t03/12/2025 19:41:32 [ERROR] [Topology 1: Removing a Port from a started Topology is not permitted.] \n\t03/12/2025 19:41:34 [ERROR] [Please Stop Device Group before removing Static LAG.] \n\t03/12/2025 19:41:34 [ERROR] [Ethernet 1(LAG) 1: Removing a Protocol from a started Protocol stack is not permitted.] \n\t03/12/2025 19:37:44 [WARNING] [Traffic Generate required] The Traffic Item was modified. Please perform a Traffic Generate to update the associated traffic Flow Groups\n\t03/12/2025 19:40:57 [WARNING] [Regenerate Traffic Item Required] The Traffic Item configuration has changed and it needs to be regenerated before applying it to the Ixia hardware","children":[{"name":"test_static_and_lacp_lag","uid":"f678c2b05adfc306","parentUid":"a5451e532acec7dd81c7db4603913a52","status":"failed","time":{"start":1741764418062,"stop":1741764425874,"duration":7812},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"a5451e532acec7dd81c7db4603913a52"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 276, in set_config\n    self.config_ixnetwork(config)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 361, in config_ixnetwork\n    self.lag.config()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/lag.py\", line 102, in config\n    self._protocol_config()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/lag.py\", line 311, in _protocol_config\n    ixn_static.remove()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/lag/lagportstaticlag_91864be5b58104e6349e12f415e28075.py\", line 259, in remove\n    self._delete()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 441, in _delete\n    raise e\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 438, in _delete\n    self._connection._delete(url)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 286, in _delete\n    return self._send_recv(\"DELETE\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 596, in _send_recv\n    self._process_response_status_code(url, headers, response)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 476, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n Unable to delete /lag:6/protocolStack/ethernet:1/lagportstaticlag:1 Ethernet 1(LAG) 1: Removing a Protocol from a started Protocol stack is not permitted.\n\tCurrent Server Errors/Warnings:\n\t03/12/2025 19:38:00 [WARNING] [Protocols Version Mismatch] Protocols Version Mismatch.\nIxNetwork Protocols Version: 10.80.2412.12\nChassis Protocols Version: 10.50.2410.2\n\t03/12/2025 19:41:23 [ERROR] [Topology 1: Removing a Port from a started Topology is not permitted.] \n\t03/12/2025 19:41:26 [ERROR] [Please Stop Device Group before removing Static LAG.] \n\t03/12/2025 19:41:26 [ERROR] [Ethernet 1(LAG) 1: Removing a Protocol from a started Protocol stack is not permitted.] \n\t03/12/2025 19:37:44 [WARNING] [Traffic Generate required] The Traffic Item was modified. Please perform a Traffic Generate to update the associated traffic Flow Groups\n\t03/12/2025 19:40:57 [WARNING] [Regenerate Traffic Item Required] The Traffic Item configuration has changed and it needs to be regenerated before applying it to the Ixia hardware","children":[{"name":"test_lacp_lag","uid":"74e5931460e50eba","parentUid":"d36398e0ab940df603fc2a301956eb2a","status":"failed","time":{"start":1741764418062,"stop":1741764447098,"duration":29036},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"d36398e0ab940df603fc2a301956eb2a"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 276, in set_config\n    self.config_ixnetwork(config)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 374, in config_ixnetwork\n    self._ixnetwork.StartAllProtocols()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/ixnetwork.py\", line 2862, in StartAllProtocols\n    return self._execute(\"startAllProtocols\", payload=payload, response_object=None)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 452, in _execute\n    response = self._connection._execute(url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 289, in _execute\n    return self._send_recv(\"POST\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 572, in _send_recv\n    return self._poll()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 640, in _poll\n    return self._process_response_status_code(\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 476, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n  Port 'novus1-715849.ccu.is.keysight.com;1;2' in the Port Group 'p2' already used by Port Group 'rxp1'","children":[{"name":"test_flow_tracking_stats","uid":"76e428223e0f11b4","parentUid":"8da827c503d6e382871794c12e4ee1fb","status":"failed","time":{"start":1741764418062,"stop":1741764454269,"duration":36207},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"8da827c503d6e382871794c12e4ee1fb"},{"name":"assert 400 == 500","children":[{"name":"test_device_connection","uid":"2d44d13ea8771d6d","parentUid":"5dc59fb3fa66403244a17904c0e00b01","status":"failed","time":{"start":1741764418062,"stop":1741764458997,"duration":40935},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"5dc59fb3fa66403244a17904c0e00b01"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 276, in set_config\n    self.config_ixnetwork(config)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 374, in config_ixnetwork\n    self._ixnetwork.StartAllProtocols()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/ixnetwork.py\", line 2862, in StartAllProtocols\n    return self._execute(\"startAllProtocols\", payload=payload, response_object=None)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 452, in _execute\n    response = self._connection._execute(url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 289, in _execute\n    return self._send_recv(\"POST\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 572, in _send_recv\n    return self._poll()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 640, in _poll\n    return self._process_response_status_code(\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 476, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n  Port 'novus1-715849.ccu.is.keysight.com;1;2' in the Port Group 'rx' already used by Port Group 'rxp1'","children":[{"name":"test_traffic","uid":"38f0136c210f7bb3","parentUid":"aee0994557962bee39e934de6ae4010e","status":"failed","time":{"start":1741764418062,"stop":1741764435712,"duration":17650},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"aee0994557962bee39e934de6ae4010e"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 584, in get_metrics\n    response = self.protocol_metrics.results(request)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/protocolmetrics.py\", line 445, in results\n    return self._filter_stats(protocol)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/protocolmetrics.py\", line 418, in _filter_stats\n    return self._get_per_device_group_stats(protocol)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/protocolmetrics.py\", line 314, in _get_per_device_group_stats\n    raise Exception(msg)\n \n                Could not retrive drill down view                 at row index 0 Failed to get the View.href property as the View object has no encapsulated resources. Either the View.find() method was not called or it failed to retrieve any resources from the server.","children":[{"name":"test_encrypt","uid":"d749d93f6df4a5f4","parentUid":"c30c85a6c46c7e04bf2115be7873f128","status":"failed","time":{"start":1741764418062,"stop":1741764471550,"duration":53488},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"test_encrypt_with_mka","uid":"8d4c9b8a224ac9cd","parentUid":"c30c85a6c46c7e04bf2115be7873f128","status":"failed","time":{"start":1741764418062,"stop":1741764521136,"duration":103074},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"c30c85a6c46c7e04bf2115be7873f128"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}]}